app/components/Logs/index.js

```diff
@@ -12,7 +12,7 @@ import { Colors, Typography } from '_styles';
 const renderLog = ({ item }) => (
   <View>
     <Text style={styles.logStyle}>
-      {`${item.timestamp} ${item.appVersion} > ${item.referenceID? item.referenceID: ''} ${item.statusCode? item.statusCode: ''} ${item.message}`}
+      {`${(item.timestamp).toLocaleString()} ${item.appVersion} > ${item.referenceID? item.referenceID: ''} ${item.statusCode? item.statusCode: ''} ${item.message}`}
```

Comment:
We have some functionality here to format Date values for the current locale: https://github.com/Plant-for-the-Planet-org/treemapper/blob/d097bc9dfea536bece4e3803adf85b98d0b67da8/app/languages/languages.js#L33 - maybe we want to apply it here to?

Reply:
changed it